pkg/runner/cluster_swarm.go

```diff
@@ -155,12 +155,47 @@ func (*ClusterSwarmRunner) Run(input *api.RunInput) (*api.RunOutput, error) {
 		return nil, fmt.Errorf("testground-redis service doesn't exist in the swarm cluster; aborting")
 	}
 
+	// Create the network.
+
+	log.Infow("creating network", "name", sname)
+
+	networkSpec := types.NetworkCreate{
+		Driver:         "overlay",
+		CheckDuplicate: true,
+		EnableIPv6:     true, // TODO: params?
+		Internal:       true,
+		Attachable:     true,
+		Scope:          "swarm",
+		Labels: map[string]string{
+			"testground.plan":     input.TestPlan.Name,
+			"testground.testcase": testcase.Name,
+			"testground.runid":    input.RunID,
+			"testground.name":     "default", // default name. TODO: allow multiple networks.
```

Comment:
This is the name that tests and the sidecar will use. (we may support more network interfaces in the future)